src/Components/SmartComponents/Reports/ReportsPage.js

```diff
@@ -20,7 +20,7 @@ const ReportsPage = () => {
                     <GridItem span={3}>
                         <Card>
                             <CardTitle>
-                                <ChartPieSolid />
+                                <ChartPieSolid style={{ verticalAlign: '-0.25rem' }}/>
```

Comment:
since you created the `styles.js` let's start adding and organize the style there.